core-web/libs/portlets/dot-locales/portlet/src/lib/dot-locales-list/dot-locales-list.component.html

```diff
@@ -10,7 +10,7 @@
                 pInputText />
         </span>
         <button
-            (click)="store.openAddEditDialog()"
+            (click)="store.openAddEditDialog(null)"
```

Comment:
Do you know if we can pass `empty`? `tore.openAddEditDialog()`

You can also set `null` as the default value:

```js
openAddEditDialog = ( value = null ) => {
// ...
}
```